packages/indexer-client/src/types/serverModelTypes.ts

```diff
@@ -285,3 +285,24 @@ export interface IndexerServerMaker {
   address: string;
   data: IndexerServerMakerData[];
 }
+
+export interface IndexerServerLeaderboardPosition {
```

Comment:
Not for this PR, but can we add account value to be returned here?

Reply:
@frankfka is this a must have? we currently don't keep track of account value in this context.

Reply:
Hmm Conor asked for it - but I thought for ROI calc, you would need account value at last update time?

Reply:
@frankfka this is addressed